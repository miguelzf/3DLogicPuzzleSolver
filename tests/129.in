- - - - - -
- - - - - -
- - P - - -
- - - - - -
- - - - - -
- - - Y - R

- - - - - -
- - - - - Y
- - - - - -
- - - - - -
- G B - - R
- - - - - -

- - - - - -
P - L X X -
- - - X X -
- - - X X -
B - G X X -
L - - - - -
