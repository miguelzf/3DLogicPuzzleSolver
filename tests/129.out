
Solution:
p  p  p  p  p  p  
p  -  -  -  -  p  
p  p  P  -  -  p  
r  r  r  r  r  p  
r  -  -  -  r  p  
r  y  y  Y  r  R  
================
r  y  -  -  -  -  ||  -  p  l  l  l  l  
r  y  y  y  y  Y  ||  P  p  L  X  X  l  
r  g  g  g  g  g  ||  g  g  g  X  X  l  
r  g  -  -  b  b  ||  b  -  g  X  X  l  
r  G  B  b  b  R  ||  B  -  G  X  X  l  
r  r  r  r  r  r  ||  L  l  l  l  l  l  
